Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:carbon-alpine
+
+MAINTAINER Apiary <sre@apiary.io>
+
+ARG DREDD_VERSION=latest
+RUN echo ${DREDD_VERSION}
+RUN apk add --no-cache make gcc g++ python git
+RUN npm config set loglevel error
+RUN npm install dredd@${DREDD_VERSION}
+RUN apk del make gcc g++ python
```

Comment:
One tip to make the docker image much smaller would be to mimimise the amount of layers it has. Each run invocation creates a seperate layer in the image, so even though you are deleting the packages they are still in previous two layer. You can make all the commands happen in a single layer to prevent this. For example:

```
RUN apk add --no-cache make gcc g++ python git && \
  npm config set loglevel error && \
  npm install dredd@${DREDD_VERSION} && \
  apk del make gcc g++ python
```

Reply:
https://github.com/apiaryio/dredd-docker/pull/2